### DISCUSSION TITLE
Will this be available for Windows?

### DISCUSSION THREAD

**Discussion by Tureti** [FDH2/UxPlay#11] [General]
Is it possible to bring out a Windows Version?
I switched form Ubuntu to Windows and there a no good Windows alternatives.

**Comment by Bernadette-Mohr** [comment]
https://letsview.com/mirror-iphone-ipad-windows-10.html
Did you try any of those options? I had read somewhere that the QuickTime player was supposed to be able to mirror an iPad screen, so I did a quick Google and this was literally the first hit. I somehow can't imagine that there is no solution to mirror iOS to Windows.

**Reply by Tureti** [reply]
Yes there are a lot and I tried one but you need to pay for a lot of them and the one I tried was bad as well...
I'll maybe try some more but I was very happy with Uxplay so...

**Reply by Tureti** [reply]
Btw I just tried one Software which is supposed to be the best according to your link and it has a 5 second delay and lags a lot...

**Comment by fduncanh** [comment]
@Tureti 

There is code for windows present in the airplay library in UxPlay, from RPiPlay,  from earlier work..
There is a fork of RPiPlay / UxPlay that appears to claim to work on Windows.

If this really works, it could be upgraded to make all features of   uxplay 1.36 work on windows i guess.
I got hints for making UxPlay work on macOS from a fork of wegank's work.
(The most dificult part was not the UxPlay code but getting CMakeLists.txt working for MacOS builds, using
the various ways of installing GStreamer on Mac ( MacPorts, Brew or direct from GStreamer's site.)
Windows might be similar.

Look at
https://github.com/wegank/ludimus

try it and see.

**Comment by Tureti** [comment]
Yes I just tried it and it actually works well.
It would be great if you or anyone found time to look into how they did it because even though it works, it is no longer active and it reminds more of the UxPlay from Years Ago.

**Comment by fduncanh** [comment]
@Tureti 

uxplay 1.56 (GITHUB VERSION) now build and runs on Windows....
https://github.com/FDH2/UxPlay

**Comment by nickcacci** [comment]
Excuse me, it is not really clear for me if right now (oct 2025) windows users should compulsory compile it from scratch or are there any ways to use precompiled binaries and get it to work on windows?

**Reply by fduncanh** [reply]
recommend you compile from scratch.    Its explained step by step in the README.  and you get latest code from github.
https://github.com/FDH2/UxPlay#building-uxplay-on-microsoft-windows-using-msys2-with-the-mingw-64-compiler

There are some "unofficial"  third-party compiled  Windows binaries out there to be found.    For example:

https://github.com/leapbtw/uxplay-windows

Looks like this  has some added Windows integration, but we haven't checked it or endorsed it.    This is free (GPLv3) software, so anyone is free to redistribute it under the terms of the GPLv3 license.   We don't distribute binaries, and leave that to others.    Many Linux distributions offer binaries.
